### DISCUSSION TITLE
Submit to F-Droid repository

### DISCUSSION THREAD

**Discussion by ghost** [advplyr/audiobookshelf#727] [Ideas]
F-Droid is a FOSS and privacy respecting alternative to Google Play - there is only open source software on it, too. Maybe it's worth adding Android app to there?

**Comment by advplyr** [comment]
There is discussion of this on the mobile app repo. https://github.com/advplyr/audiobookshelf-app/discussions/171
I'm not sure how far it got or what the roadblocks were on that, but it was also being discussed in the Discord.
